### PR TITLE
Quick fixes on auto uisi RS

### PR DESCRIPTION
**Pull request by BillCarsonFr**
Quick fixes on auto rageshake lab feature:

-> Align name and format with web (snake case, use json)
-> Fix an issue with auto rageshake disabling itself on restart
-> Update the report server app name (so that it goes to a dedicated private repo)

Depends on https://github.com/matrix-org/matrix-ansible-private/pull/4247 (do not merge before)

### CODE REVIEW

File: vector/src/main/java/im/vector/app/AutoRageShaker.kt [hunk 1]
```diff
@@ -68,6 +68,7 @@ class AutoRageShaker @Inject constructor(
     fun initialize() {
         observeActiveSession()
         // It's a singleton...
+        enable(vectorPreferences.labsAutoReportUISI())
```
Comment (bmarty) [inline vector/src/main/java/im/vector/app/AutoRageShaker.kt:71]: Please do not insert new code between a line of code and its comment :)

File: vector/src/main/java/im/vector/app/features/rageshake/BugReporter.kt [hunk 2]
```diff
@@ -259,13 +259,23 @@ class BugReporter @Inject constructor(
...
+                                when (reportType) {
+                                    ReportType.AUTO_UISI_SENDER,
+                                    ReportType.AUTO_UISI           -> {
+                                        addFormDataPart("app", "element-auto-uisi")
+                                    }
+                                    else -> {
+                                        addFormDataPart("app", "riot-android")
+                                    }
+                                }
+                            }
```
Comment (bmarty) [inline vector/src/main/java/im/vector/app/features/rageshake/BugReporter.kt:278]: OK... You already expect me to say something about that, no? :)

**Review by bmarty — COMMENTED** [review]
Nice, thanks.
We wait to merge this PR.

File: vector/src/main/java/im/vector/app/AutoRageShaker.kt [hunk 3]
```diff
@@ -221,17 +224,19 @@ class AutoRageShaker @Inject constructor(
                 withCrashLogs = true,
                 withKeyRequestHistory = true,
                 withScreenshot = false,
-                theBugDescription = "UISI detected $matchingIssue",
+                theBugDescription = "Auto-reporting decryption error \nRecipient rageshake: $matchingIssue",
```
Comment (duxovni) [inline vector/src/main/java/im/vector/app/AutoRageShaker.kt:227]: If we want this to be totally consistent with element-web, it'd be
```suggestion
                theBugDescription = "Auto-reporting decryption error (sender)\nRecipient rageshake: $matchingIssue",
```
but that's not a huge deal, and we have the labels now in any case so maybe element-web doesn't need this either anymore.

**Review by duxovni — CHANGES_REQUESTED** [review]
Otherwise LGTM in terms of consistency with element-web.

File: vector/src/main/java/im/vector/app/features/rageshake/BugReporter.kt (hunk 2 above)
Comment (bmarty) [reply vector/src/main/java/im/vector/app/features/rageshake/BugReporter.kt:278]: I mean having something like`.addFormDataPart("app", getRageshakeApp(reportType))` would be nicer

Comment (BillCarsonFr) [reply vector/src/main/java/im/vector/app/features/rageshake/BugReporter.kt:278]: you read my mind, or did I read yours?

File: vector/src/main/java/im/vector/app/AutoRageShaker.kt (hunk 3 above)
Comment (BillCarsonFr) [reply vector/src/main/java/im/vector/app/AutoRageShaker.kt:227]: updated

File: vector/src/main/java/im/vector/app/features/rageshake/BugReporter.kt (hunk 2 above)
Comment (BillCarsonFr) [reply vector/src/main/java/im/vector/app/features/rageshake/BugReporter.kt]: done

File: vector/src/main/java/im/vector/app/AutoRageShaker.kt (hunk 1 above)
Comment (BillCarsonFr) [reply vector/src/main/java/im/vector/app/AutoRageShaker.kt:70]: done

**Review by bmarty — APPROVED** [review]
Thanks for the update!

**Pull request merged**